src/Microsoft.VisualStudio.Web.CodeGeneration.Design/Program.cs

```diff
@@ -41,6 +41,10 @@ private static void Execute(string[] args, ConsoleLogger logger)
             var noBuild = app.Option("--no-build", "", CommandOptionType.NoValue);
             var simMode = app.Option("--simulation-mode", Resources.SimulationModeOptionDesc, CommandOptionType.NoValue);
 
+            while (!System.Diagnostics.Debugger.IsAttached)
+            {
+            }
+
```

Comment:
Whoops, shouldn't have added this to the PR. I'll be sure to remove it before a real check-in.